src/server/views/partials/menu-left.html

```diff
@@ -12,20 +12,16 @@
   <li class="list-group-item pointer" data-config="true" data-manager-show="manager-files">
     {{@root.text.navigation}}
   </li>
-  {{#if @root.config.users.enable}}
-    {{#isAuthorized '/abe/references' @root.user.role.workflow}}
+  {{#isAuthorized '/abe/references' @root.user.role.workflow}}
   <li class="list-group-item pointer" >
     <a href="/abe/reference">edit references</a>
   </li>
-    {{/isAuthorized}}
-  {{/if}}
-  {{#if @root.config.users.enable}}
-    {{#isAuthorized '/abe/structure' @root.user.role.workflow}}
+  {{/isAuthorized}}
+  {{#isAuthorized '/abe/structure' @root.user.role.workflow}}
```

Comment:
this is not what I see. What I see is that only a user's role can ENABLE these buttons.